src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-hex/Format-Hex.cs

```diff
@@ -308,18 +318,21 @@ private void ProcessObjectContent(PSObject inputObject)
         #region Output
 
         /// <summary>
-        /// Outputs the hexadecimial representaion of the of the input data.
+        /// Outputs the hexadecimial representaion of the input data.
         /// </summary>
-        /// <param name="inputBytes"></param>
-        /// <param name="path"></param>
-        /// <param name="offset"></param>
-        private void ConvertToHexidecimal(byte[] inputBytes, string path, UInt32 offset)
+        /// <param name="inputBytes">Bytes for the hexadecimial representaion.</param>
+        /// <param name="path">File path.</param>
+        /// <param name="offset">Offset in the file.</param>
+        private void ConvertToHexidecimal(Span<byte> inputBytes, string path, Int64 offset)
```

Comment:
Since this method does the actual `WriteObject()`, perhaps rename as `WriteHexidecimal()`?

Reply:
Fixed.